linode/sshkey/framework_resource_schema.go

```diff
@@ -34,9 +33,6 @@ var frameworkResourceSchema = schema.Schema{
 		"id": schema.Int64Attribute{
 			Description: "The unique identifier for this SSH key.",
 			Computed:    true,
-			PlanModifiers: []planmodifier.Int64{
-				int64planmodifier.UseStateForUnknown(),
-			},
```

Comment:
Was this causing some trouble?

Reply:
Good catch! Looks like this was just an irrelevant change that I forgot to stash. I'll push up a commit to revert it.